tests/seed/commerce/test_agreement.py

```
@@ -0,0 +1,111 @@
+    assert result is None
+
+
+def test_set_agreement(context, agreement) -> None:
+    result = init_agreement(agreement, context=context)
+
+    assert result == agreement
+    assert context["commerce.agreement.id"] == agreement.id
+    assert context.get_resource("commerce.agreement", agreement.id) == agreement
+
+
+def test_build_agreement(context) -> None:  # noqa: WPS218
```

Ruff (0.14.7)

Unused `noqa` directive (unknown: `WPS218`)

Remove unused `noqa` directive

(RUF100)